pkg/config/rhssoUser.go

```diff
@@ -47,3 +49,16 @@ func (r *RHSSOUser) Validate() error {
 	}
 	return r.ValidateCommon()
 }
+
+func (r *RHSSOUser) GetReplicasConfig(inst *integreatlyv1alpha1.RHMI) int {
+	numberOfReplicas := 2
+
+	if testResources.RunningInProw(inst) {
+		numberOfReplicas = 1
+	}
+	if inst.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
+		numberOfReplicas = 3
+	}
+
+	return numberOfReplicas
+}
```

Comment:
```suggestion
func (r *RHSSOUser) GetReplicasConfig(inst *integreatlyv1alpha1.RHMI) int {
	if testResources.RunningInProw(inst) {
		return 1
	}
	if inst.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
		return 3
	}

	return 2
}
```

I noticed that when running in prow it could still set 3 as the number of replicas because it doesn't return after the `if` when the installation type is Managed API